tests/model_selection/test_hyperband_non_daskarray.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import pandas as pd
+
+from dask_ml.model_selection import HyperbandSearchCV
+from dask_ml.datasets import make_classification
+from distributed.utils_test import gen_cluster
+from sklearn.linear_model import SGDClassifier
+
+
+@gen_cluster(client=True)
+def test_pandas(c, s, a, b):
+    X, y = make_classification(chunks=100)
+    X, y = pd.DataFrame(X.compute()), pd.Series(y.compute())
```

Comment:
Looks like the test is failing in the `.compute` function. Why not use `from sklearn.datasets import make_classification`?

Reply:
It's strange, it was working on my local machine.
Let me try to change it. Maybe it's something to do with how I wrote the test function. I just tried to emulate the structure I found in `test_hyperband.py`